.vscode/launch.json

```diff
@@ -0,0 +1,64 @@
+{
+    "configurations": [
+        {
+            "comments": [
+                "Full launch.json configuration details can be found here:",
+                "https://code.visualstudio.com/docs/cpp/launch-json-reference"
+            ],
+            "name": "DEBUG: (gdb-oneapi) nbody_dpcpp_d Launch",
+            "type": "cppdbg",
+            "request": "launch",
+            "preLaunchTask": "Debug C/C++: DPCPP Makefile",
+            "postDebugTask": "",
+            "program": "/home/pablo/src/n-body/cuda-to-sycl-nbody/nbody_dpcpp_d",
```

Comment:
hould this reference a specific path on a machine? Is there a way to make this generic?